core/server/src/main/java/alluxio/master/AlluxioMasterRestServiceHandler.java

```diff
@@ -95,19 +100,26 @@ public AlluxioMasterRestServiceHandler(@Context ServletContext context) {
 
   /**
    * @summary get the Alluxio master information
+   * @param rawConfiguration if it's true, raw configuration values are returned,
+   *    otherwise, they are looked up; if it's not provided in URL queries, then
+   *    it is null, which means false.
    * @return the response object
    */
   @GET
   @Path(GET_INFO)
-  public Response getInfo() {
+  public Response getInfo(@QueryParam(QUERY_RAW_CONFIGURATION) final Boolean rawConfiguration) {
```

Comment:
is there a use case for not wanting the configuration values to be resolved? I wonder if we could simplify things by just always resolving them.